posthog/hogql_queries/test/test_actors_query_runner.py

```diff
@@ -73,12 +73,11 @@ def test_default_persons_query(self):
             select=[
                 ast.Field(chain=["id"]),
                 ast.Field(chain=["id"]),
-                ast.Field(chain=["created_at"]),
                 ast.Constant(value=1),
             ],
             select_from=ast.JoinExpr(table=ast.Field(chain=["persons"])),
             where=None,
-            order_by=[ast.OrderExpr(expr=ast.Field(chain=["created_at"]), order="DESC")],
+            order_by=[ast.OrderExpr(expr=ast.Field(chain=["id"]), order="ASC")],
```

Comment:
do we need to have an `order_by` clause for just ids?
Guessing we add it by default [here](https://github.com/PostHog/posthog/pull/26623/files?file-filters%5B%5D=.py&file-filters%5B%5D=.ts&show-viewed-files=true#diff-5af03d6d5feabb17ff9fa4c2f8ce44cb4d47e32efb70d0cad761e3961ccf1f06R248)